polygon.js

```diff
@@ -194,10 +188,25 @@ async function getUnbondNonces(address) {
         throw new Error(error);
     }
 }
+async function getCurrentEpoch() {
+    try {
+        return await contract_staking.methods.currentEpoch().call();
+    } catch (error) {
+        throw new Error(error);
```

Comment:
could be just
```
async function getCurrentEpoch() {
    return await contract_staking.methods.currentEpoch().call();
}
```

rethrowing the error leads to the loss of the stack trace of the original error. It's better to let the error propagate up the call stack. Not a blocker.